### DISCUSSION TITLE
Why organization members/co-laborators can't only have access to edit repository description and tags?

### DISCUSSION THREAD

**Discussion by Blueberryy** [community/community#27320] [Other Feature Feedback, Questions, & Ideas]
I want to make someone to have access to change repository description or add more cool tags to it. Some repository roles be added to github?

**Comment by sawyerbivens** [comment]
Not sure if this is what you mean but I believe there's some kind of way for people you [invite to collaborate](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-access-to-your-personal-repositories/inviting-collaborators-to-a-personal-repository) to modify the repo's tags as theres [this page](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/configuring-tag-protection-rules) on the Github docs about protected tags in a project. I'm not too sure on whether or not invited collaborators can change things like the project description though.


However if you're requesting a feature for them to add roles to projects I would create that discussion in the [Discussions category](https://github.com/orgs/community/discussions/categories/discussions) so more people can give feedback on the idea.
